GrandCommonDivisor.Core/Models/NumberParseResult.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace GrandCommonDivisor.Core.models
```

Comment:
Namespace name's parts should start with capital letter

Reply:
Fixed.